frontend/src/modules/entitieslist/components/EntitiesList.tsx

```diff
@@ -311,21 +311,20 @@ export const EntitiesListBase: React.FC<InternalProps> = (
         );
     };
 
-    // InfiniteScroll will display information about loading during the request
 
+    // InfiniteScroll will display information about loading during the request
     const hasMore = props.entities.fetching || props.entities.hasMore;
+    const [sentryRef, { rootRef }] = useInfiniteScroll({
+        loading: props.entities.fetching,
+        hasNextPage: hasMore,
+        onLoadMore: getMoreEntities,
+        rootMargin: '0px 0px 400px 0px',
+    });
+
+
     return (
-        <div className='entities unselectable' ref={list}>
-            <InfiniteScroll
-                pageStart={1}
-                loadMore={getMoreEntities}
-                hasMore={hasMore}
-                loader={
-                    <SkeletonLoader key={0} items={props.entities.entities} />
-                }
-                useWindow={false}
-                threshold={600}
-            >
+        <div className='entities unselectable' ref={rootRef}>
+            <div ref={list}>
```

Comment:
Can we drop the `list` reference and only use `rootRef` (even for the "Scroll to selected entity" use case)?

Reply:
It can't be dropped as far as I have tried,when I removed it, the application didn't work correctly, i had to include it.. By the way it's not hurting any changes, so I suggest we leave it.

Reply:
> It can't be dropped as far as I have tried,when I removed it, the application didn't work correctly, i had to include it.. By the way it's not hurting any changes, so I suggest we leave it.

it's now working without the list ref. I have also removed any code related to it . :)